openbb_terminal/cryptocurrency/coinbase_advanced_helpers.py

```diff
@@ -0,0 +1,423 @@
+        }
+        # Handle parsing for diff ordee types
+        _order_config = orders["order_configuration"]
+
+        if "market_market_ioc" in _order_config:
+            _flat_response["order_type"] = "market_market_ioc"
+            _flat_response["quote_size"] = _order_config["market_market_ioc"][
+                "quote_size"
+            ]
+            _flat_response["base_size"] = _order_config["market_market_ioc"][
+                "base_size"
+            ]
+
+        if "limit_limit_gtc" in _order_config:
+            _flat_response["order_type"] = "limit_limit_gtc"
+            _flat_response["limit_price"] = _order_config["limit_limit_gtc"][
+                "limit_price"
+            ]
+            _flat_response["base_size"] = _order_config["limit_limit_gtc"]["base_size"]
+            _flat_response["post_only"] = _order_config["limit_limit_gtc"]["post_only"]
+
+        if "limit_limit_gtd" in _order_config:
+            _flat_response["order_type"] = "limit_limit_gtd"
+            _flat_response["base_size"] = _order_config["limit_limit_gtd"]["base_size"]
+            _flat_response["limit_price"] = _order_config["limit_limit_gtd"][
+                "limit_price"
+            ]
+            _flat_response["end_time"] = _order_config["limit_limit_gtd"]["end_time"]
+            _flat_response["post_only"] = _order_config["limit_limit_gtd"]["post_only"]
+
+        if "stop_limit_stop_limit_gtc" in _order_config:
+            _flat_response["order_type"] = "stop_limit_stop_limit_gtc"
+            _flat_response["base_size"] = _order_config[_flat_response["order_type"]][
+                "base_size"
+            ]
+            _flat_response["limit_price"] = _order_config[_flat_response["order_type"]][
+                "limit_price"
+            ]
+            _flat_response["stop_price"] = _order_config[_flat_response["order_type"]][
+                "stop_price"
+            ]
+            _flat_response["stop_direction"] = _order_config[
+                _flat_response["order_type"]
+            ]["stop_direction"]
+
+        if "stop_limit_stop_limit_gtd" in _order_config:
+            _flat_response["order_type"] = "stop_limit_stop_limit_gtd"
+            _flat_response["base_size"] = _order_config[_flat_response["order_type"]][
+                "base_size"
+            ]
+            _flat_response["limit_price"] = _order_config[_flat_response["order_type"]][
+                "limit_price"
+            ]
+            _flat_response["stop_price"] = _order_config[_flat_response["order_type"]][
+                "stop_price"
+            ]
+            _flat_response["end_time"] = _order_config[_flat_response["order_type"]][
+                "end_time"
+            ]
+            _flat_response["end_time"] = _order_config[_flat_response["order_type"]][
+                "end_time"
+            ]
+
+        flat_response.append(_flat_response)
+
+    df = pd.DataFrame(flat_response)
+    if df.empty:
```

Comment:
I think this is causing my problem.  You may need to define the columns like you do above?
```
columns=[
                "product_id",
                "side",
                "price",
                "size",
                "order_type",
                "created_time",
                "status",
            ]
```

Reply:
These empty dataframes also need an order_id column

Reply:
@jmaslek What is the command sequence that raises the error?

Reply:
This happens when I just try entering the menu.  its from the argument choices (which we use to autogenerate the autocomplete)

Reply:
I see now. The choices for the argument order_id  get loaded from a CB query. The choices get generated when you first enter the menu. I retrieve all the orders first and then filter. I think that is why it works for me, I have some orders. I am rethinking this, I will make it a string and you have to paste the order_id.